src/sentry/event_manager.py

```diff
@@ -863,6 +867,11 @@ def _get_or_create_release_many(jobs: Sequence[Job], projects: ProjectsMapping)
         if not job["release"]:
             continue
 
+        if options.get("performance.issues.create_issues_through_platform", False) and projects[
+            job["project_id"]
+        ].get_option("sentry:performance_issue_create_issue_through_platform", False):
+            continue
```

Comment:
> We don't need to filter like this in _get_or_create_release_many and _get_or_create_environment_many - transactions should create those regardless of whether they create groups. We just need to filter in _get_or_create_group_environment_many and _get_or_create_group_release_many.

oops, added correct checks

> It might actually be enough to just filter in _save_aggregate_performance, since those other functions operate only on the groups provided.

I've added a check just in case to be sure